src/ui/macstr_ui.py

```diff
@@ -0,0 +1,201 @@
+import sys
+import yaml
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+
+class Ui_MainWindow(object):
+    def setupUi(self, MainWindow):
+
+        # loading in the JSON data
+        with open("../../notebooks/test_image_coords.json", "r") as contents:
```

Comment:
The `test_image_coords.json` was shown in a previous PR. https://github.com/WayScience/macrophage-striation-analysis/pull/1. This will be used to select and mark image.

Reply:
Got it, thanks, and apologies for missing this!